routers/cities.py

```
@@ -0,0 +1,60 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from session_db import db_session
+import crud.cities
+import schemas.cities
+
+
+router = APIRouter()
+
+
+@router.get("/city/", response_model=list[schemas.cities.GetCity])
+def get_cities_all(db: Session = Depends(db_session)):
+    return crud.cities.get_all_cities(db)
+
+
+@router.get("/city/{city_id}", response_model=schemas.cities.GetCity)
+def get_cities_id(city_id: int, db: Session = Depends(db_session)):
+    return crud.cities.get_city_by_id(db, city_id)
```

This endpoint doesn't handle the case where a city with the given ID is not found. `crud.cities.get_city_by_id` will return `None`, causing the endpoint to respond with a `200 OK` and a `null` body. You should add a check for the result and raise an `HTTPException` with a `404 Not Found` status, similar to what you've done in the `delete_city` and `update_city` functions.